src/spikeinterface/extractors/neoextractors/neobaseextractor.py

```diff
@@ -462,7 +462,7 @@ def _infer_sampling_frequency_from_analog_signal(self, stream_id: Optional[str]
                 f"sampling_frequency = {sampling_frequency} \n"
```

Comment:
No need to change, but did you know about: 

```python
f"{stream_id=}"
```

This blew my mind when I found out about that.

Reply:
Yeah, I wish I could use it but is not in the oldest version of python that we support.

Reply:
Ah, it seems that it is python 3.8. God to know, I will start to use.

Reply:
https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging

Reply:
Sweet. I was thinking I saw it somewhere in the Neo code base (where I discovered it) and Neo also supports 3.8 so I was about to have a freakout needing to track down where I saw it there.